packages/react-composites/src/composites/MeetingComposite/state/MeetingParticipants.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { RemoteParticipantState } from '@internal/calling-stateful-client';
+import { CommunicationIdentifier } from '@azure/communication-common';
+import { MeetingEndReason } from './MeetingEndReason';
+
+export interface MeetingParticipant
+  extends Pick<RemoteParticipantState, 'displayName' | 'state' | 'videoStreams' | 'isMuted' | 'isSpeaking'> {
+  id: CommunicationIdentifier;
+  meetingEndReason?: MeetingEndReason;
+}
+
+export function createMeetingParticipant(callParticipant: RemoteParticipantState): MeetingParticipant {
+  const { displayName, state, videoStreams, isMuted, isSpeaking, identifier, callEndReason } = callParticipant;
+
+  return { id: identifier, meetingEndReason: callEndReason, displayName, state, videoStreams, isMuted, isSpeaking };
+}
+
+export function convertCallParticipantsToMeetingParticipants(callParticipants: {
```

Comment:
`convert*` is ok for this, simply be consistent in naming all the conversion functions you have. Your choice of `mergeX` and `convertX` is nice. Let's just double check that they're used consistently (e.g., the one above should be `convertChatParticipantToMeetingParticipant` following this convetion)

Reply:
Updating to `meetingParticipantsFromCallParticipants` to follow above naming